alf/layers.py

```diff
@@ -205,7 +202,7 @@ def _kron(A, B):
                 h = torch.cat((h_n, h_i), dim=1)
                 return h
 
-            B = _get_haar_matrix(n)
+            B = _get_haar_matrix(n) / torch.sqrt(torch.Tensor([n]))
```

Comment:
The term ``/ torch.sqrt(torch.Tensor([n]))`` was not included before as it is a global scaling. Now added to make it fully compatible with explicit Haar matrices.